cachito/web/validation.py

```diff
@@ -46,3 +49,55 @@ def validate_dependency_replacements(replacements: List[Dict[str, Any]]) -> None
         raise ValidationError('"dependency_replacements" must be an array')
     for replacement in replacements:
         validate_replacement(replacement)
+
+
+class RequestBodyValidator(decorators.validation.RequestBodyValidator):
+    """
+    This class overrides the default connexion RequestBodyValidator.
+
+    It returns the complete string representation of the
```

Comment:
Doesn't the original exception also return the complete error message? 

I thought this change was to keep the format and exception type we expect in Cachito (ValidationError, with `{"error": "message"}` instead of `{"detail": "message"}`). 

Correct if I`m wrong, though :sweat_smile:.

Reply:
Yeah, IIRC the only change we needed was to have "error" in the JSON returned to the user (not "detail")